docs/docsite/rst/playbooks_variables.rst

```diff
@@ -884,6 +884,7 @@ Basically, anything that goes into "role defaults" (the defaults folder inside t
 
 .. [1] Tasks in each role will see their own role's defaults. Tasks defined outside of a role will see the last role's defaults.
 .. [2] Variables defined in inventory file or provided by dynamic inventory.
+.. [3] This includes other vars pulled in by 'vars plugins', host_vars/group_vars are added by the default vars plugin shipped with Ansible.
```

Comment:
Please change the comma to a semicolon

Reply:
that breaks the list and changes the meaning